bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -0,0 +1,48 @@
+# Generated via query_scheduling/generate_airflow_dags 
```

Comment:
This Jinja template is beautiful. It's surprisingly readable and gives a good sense of what the generated output is going to look like.